lib/reverse_characters.rb

```diff
@@ -1,2 +1,21 @@
+#To test my code in repl.it, created an str variable using the line below:
+#str = "Today is a good day".split("")
+
 def reverse_characters(str)
+arr_end = str.length - 1
+new_arr = Array.new
+
+(arr_end).downto(0) do |i|
+	#Create a variable to hold the array element's value for each iteration 
+	z = str[i]
+	#Add each iteration's value to the end of the array
+	new_arr.push(z)
+end
+
+#Creates the final reversed string
+fnl_arr = new_arr.join
+
+#Print value of final variable
+puts fnl_arr
```

Comment:
`puts` is a command specifically for displaying information on the command line. The actual [return](http://ruby-doc.org/core-1.9.3/IO.html#method-i-puts) of a `puts` statement is `nil`. Right now, your method is returning the value of the expression `puts fnl_arr`, which is `nil`.

How might you change line 19 so that your method returns the value of `fnl_arr`?